kubernetes/test-infra/eks/kubernetes-config/main.tf

```diff
@@ -56,14 +56,10 @@ resource "kubernetes_deployment" "test" {
   }
 }
 
-resource helm_release nginx_ingress {
-  name       = "nginx-ingress-controller"
-
-  repository = "https://charts.bitnami.com/bitnami"
-  chart      = "nginx-ingress-controller"
-
-  set {
-    name  = "service.type"
```

Comment:
I'm guessing this is the default service type now? No need to carry this value over?